src/routes/newsletters/index.xml.ts

```diff
@@ -0,0 +1,43 @@
+import { fetchNewsletters } from './_query';
+import type { EndpointOutput } from '@sveltejs/kit';
+import type { Newsletter } from './_query';
+
+const website = 'https://acmcsuf.com';
+
+const renderXML = (post: Newsletter): string => {
+  return `<item>
+  <title>${post.title}</title>
+  <description>${post.labels.join(', ')}</description>
+  <link>${website}/newsletters/${post.id}/</link>
+  <pubDate>${new Date(post.lastEdited)}</pubDate>
+  <content:encoded>
+    <div style="margin-top: 50px; font-style: italic;">
+      <strong>
+        <a href="${website}/newsletters/${post.id}">
```

Comment:
This should be a synopsis of the blog article instead of just having a "Keep reading" link. If it's outside the scope of this PR, I think we should create a new follow-up issue to track it.

Reply:
Also, is it a good idea to format RSS XML like it's HTML? HTML is much less strict; the generated output might not be conformant.

Reply:
HTML is less strict indeed, so instead, we could use a different approach than using a string template. I have not used this library personally, but [node-rss](https://github.com/dylang/node-rss) seems to be a possible choice for this purpose. The node-rss repository has not been updated in over 5 years, but that might be fine since RSS feeds are not expected to change anytime soon AFAIK.